core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -301,6 +301,8 @@ package object config {
       .doc("Number of subdirectories inside each path listed in spark.local.dir for " +
         "hashing Block files into.")
       .intConf
+      .checkValue(_ > 0, "The number of subdirectories for put hashing Block files into " +
```

Comment:
Maybe it is good to keep things consistent with other parts of the code? I see a lot of examples such as:

`.checkValue(v => v > 0, "The value should be a positive integer.")`

Reply:
Agree with above, seems the message is a bit repeated with the description in the doc.

Reply:
thanks,I change this error message

Reply:
@lcqzte10192193  The PR title should also be fixed, How about "Ensure the value of `spark.diskStore.subDirectories` to be a positive number?" or something else. And you should also put the jira number in the PR title.

Reply:
I'd put the key string in the error message, and just say "must be positive".
The title needs to be fixed here; it's the wrong way around. Add `[MINOR]` to the front. @lcqzte10192193

Reply:
thanks，I fixed it